sirepo/package_data/static/js/ml.js

```diff
@@ -1517,7 +1565,7 @@ SIREPO.app.directive('neuralNetLayersForm', function(appState, mlService, panelS
             };
 
             $scope.deleteLayer = function(idx) {
-                appState.models.neuralNet.layers.splice(idx, 1);
+                $scope.layerLevel.splice(idx, 1);
```

Comment:
(comment here because there is no good place) - If you add a layer and then cancel:
1. The UI does not revert
2. If you then try to add a new layer you get this error:
```javascript
TypeError: Cannot read properties of undefined (reading 'layer')
    at $scope.fieldTrack (ml.js?20220804.150015:1575:30)
    at fn (eval at compile (angular.js:16418:15), <anonymous>:4:313)
    at z (angular.js:33785:18)
    at angular.js:33844:25
    at angular.js:19002:13
    at m.$digest (angular.js:19140:23)
    at b.$apply (angular.js:19500:13)
    at HTMLSelectElement.<anonymous> (angular.js:35709:9)
    at sg (angular.js:3824:3)
    at HTMLSelectElement.d (angular.js:3812:9)
```

Reply:
Good catch. I'll work on that. @moellep  and I discussed some other changes as well that will be necessary for the upcoming network importing feature so I am converting this PR to draft until I have worked some stuff out.